packages/plugin-core/src/test/suite-integ/CopyNoteLink.test.ts

```diff
@@ -366,7 +366,7 @@ suite("CopyNoteLink", function () {
             engine,
           });
 
-        const editor = await openNote(noteWithTarget);
+        let editor = await openNote(noteWithTarget);
```

Comment:
was this change intentional? maybe I'm reading it wrong, but the second `openNote()` call seems redundant?

Reply:
This test was failing on github every now and then; for some reason, the editor was still sometimes reading the previous document. I added this change in hopes that it fixes that.